platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/userlocation/MyLocationTrackingModeActivity.java

```diff
@@ -261,17 +284,33 @@ public void onLowMemory() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.menu_tracking, menu);
+        dismissLocationTrackingOnGestureItem = menu.findItem(R.id.action_toggle_dismissible_location);
+        dismissBearingTrackingOnGestureItem = menu.findItem(R.id.action_toggle_dismissible_bearing);
+        enableRotateGesturesItem = menu.findItem(R.id.action_toggle_rotate_gesture_enabled);
+        enableScrollGesturesItem = menu.findItem(R.id.action_toggle_scroll_gesture_enabled);
+        setCheckBoxes();
         return true;
     }
 
+    private void setCheckBoxes() {
+        if(mapboxMap != null && dismissBearingTrackingOnGestureItem != null) {
```

Comment:
Hi @clydebarrow no worries, I will fix the comments up, 
thanks again for the contribution, awesome stuff!
Yeah the checkstyle is something we still need to integrate as part of our CI. 
We also need to make the docs on configuration more visible for external contributions.